paasta_tools/bounce_lib.py

```diff
@@ -339,19 +339,24 @@ def crossover_bounce(
     happy_new_tasks,
     old_app_live_happy_tasks,
     old_app_live_unhappy_tasks,
+    margin_factor=1.05,
```

Comment:
@oktopuz can you make this a param? I would be happy to pair with you to help hint at how to do that.